command-expansion-server/src/utils/InheritedError.ts

```diff
@@ -0,0 +1,18 @@
+export type ErrorLike = {
+  message: string,
+  name: string,
+  stack?: string | null,
```

Comment:
Silly TS question: is the `| null` really needed here? I'd assume that `stack?: <type>` means that `stack` can be `<type>` or `null` already.

In `assertions.ts` I see: `context?: T`, which is what I would expect.

Reply:
The question mark says its also possibly undefined. So you can omit it, give it a string, or give it null.

Reply:
Ohh, thanks! Makes sense